database/mysql/0015_ipm2.sql

```diff
@@ -0,0 +1,42 @@
+/*--------------------- 15 IPM2  ------------------ */
+
+SET @bios_db_schema_version = '201904250001';
+SET @bios_db_schema_filename = '0015_ipm2.sql';
+
+use box_utf8;
+
+/* This should be the first action in the SQL file */
+START TRANSACTION;
+INSERT INTO t_bios_schema_version (tag, timestamp, filename, version) VALUES('begin-import', UTC_TIMESTAMP() + 0, @bios_db_schema_filename, @bios_db_schema_version);
+
+alter table t_bios_asset_element add id_secondary varchar(255) NULL;
+
```

Comment:
The creation of the **id_secondary** column on table **t_bios_asset_link** is missing:

ALTER TABLE t_bios_asset_link ADD id_secondary varchar(255) NULL;